linux57-tkg/linux57-tkg-patches/0005-v5.7_undead-pds099o.patch

```diff
@@ -569,9 +569,9 @@ index 3bac0a8ceab2..d6d384ddb57d 100644
  #define SCHED_BATCH		3
 -/* SCHED_ISO: reserved but not implemented yet */
 +/* SCHED_ISO: Implemented in BFS/MuQSSPDS only */
-+#ifdef CONFIG_SCHED_PDS
```

Comment:
If it's okay for you I'd prefer if those changes were part of another PR.

Reply:
Okay no problem! This is another fix so Ubuntu can compile.

Reply:
I followed that despite my silence :stuck_out_tongue: But since it's not directly related to the ubuntu building implementation and more of a misc source fix, I feel like it should be separate for proper indexing.

Reply:
I guessed so! Since you were still active in this repository :sunglasses:

Reply:
Done!